### DISCUSSION TITLE
load event not working in v3

### DISCUSSION THREAD

**Discussion by ShekhSaifuddin007** [nuxt/nuxt#19452] [Questions] [answered]
I'm migrating my nuxt2 project to nuxt3, in my application many places I have used the `load` event for some operations but in v3 this `load` event not working anymore other events working like `scroll, resize` but `load` doesn't

### app.vue

```html
<template>
  <NuxtLayout />
</template>
```

### layouts/default.vue

```html
<script setup>
if (process.client) {
    window.addEventListener('load', () => {
        alert('hello');
    });
}
</script>

<template>
    <div class="w-full">
        <NavMainNav />

        <main class="h-auto">
            <slot />
        </main>
    </div>
</template>
```

**Comment by TheAlexLichter** [comment, marked as answer]
Duplicate of https://github.com/nuxt/nuxt/discussions/19198
